gateway/src/main/java/com/hurence/webapiservice/http/ingestion/IngestionApiImpl.java

```diff
@@ -35,14 +36,11 @@ public IngestionApiImpl(HistorianService service) {
     @Override
     public void importJson(RoutingContext context) {
         JsonArray getMetricsParam;
```

Comment:
This variable is not used outside the try block so it should be inside the try bloc. Always reduce at maximum the scope of your variables